src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -1506,9 +1565,15 @@ internal SignatureHelp GetSignatureHelp(TextDocumentPosition textDocumentPositio
             // if there are no completions then provide the default list
             if (resultCompletionItems == null)
             {
-                resultCompletionItems = AutoCompleteHelper.GetDefaultCompletionItems(scriptDocumentInfo, useLowerCaseSuggestions);
+                resultCompletionItems = AutoCompleteHelper.GetDefaultCompletionItems(scriptDocumentInfo, useLowerCaseSuggestions);                
             }
 
+            //invoke the completion extensions            
+            foreach(var completionExt in _completionExtensions.Values)
+            {
+                resultCompletionItems = await completionExt.HandleCompletionAsync(connInfo, scriptDocumentInfo, resultCompletionItems, new CancellationTokenSource(200).Token).ConfigureAwait(false);
```

Comment:
>200 [](start = 163, length = 3)

Suggest using a const value for this and put an explanation of why you chose this value